includes/class-wc-google-gtag-js.php

```
@@ -368,6 +368,25 @@ protected static function get_consent_modes(): array {
 		return apply_filters( 'woocommerce_ga_gtag_consent_modes', $consent_modes );
 	}
 
+	/**
+	 * Returns events filtered by name.
+	 * 
+	 * @param string $event_name Name of the event to filter.
+	 * @return array
+	 */
+	public function get_events_by_name( $event_name ) {
+		$found_events = array();
+		$all_events = $this->script_data['events'];
```

PHPStan (2.1.46)

[error] Offset 'events' does not exist on string.


(offsetAccess.notFound)